packages/terra-application-navigation/tests/wdio/application-navigation-spec.js

```diff
@@ -0,0 +1,150 @@
+Terra.describeViewports('ApplicationNavigation - Large', ['large'], () => {
```

Comment:
nit: We can drop `Large`. This will be included in the error output by the `Terra.describeViewports` helper

Reply:
Should this be testing more than the large viewport? It looks like there are some stale screenshots referencing small viewport size in this PR